apps/emqx/test/emqx_listeners_update_SUITE.erl

```diff
@@ -116,6 +116,55 @@ t_update_conf(_Conf) ->
     ?assert(is_running('wss:default')),
     ok.
 
+t_update_tcp_keepalive_conf(_Conf) ->
```

Comment:
does this test belong to this PR?

Reply:
not really, but I change parse_tcp_keepalive/1 's binary:split to to_comma_separated_binary/1
I want to make sure the parse_tcp_keeaplive's return is the same as before, so I add a new test.